File: CodeFreak1/ClientApp/src/app/event/dtos/event-performance-view-model.ts
import { UsersViewModel } from "../../Security/Dtos/users-view-model";
import { FileViewModel } from "../../Security/Dtos/file-view-model";
import { SubmissionViewModel } from "../../problem/dtos/submission-view-model";
import { RequestStatus } from "../../Security/Dtos/request-status";

export class EventPerformanceViewModel extends RequestStatus {
  public User: UsersViewModel;
  public UserImage: FileViewModel;
  public Submissions: Array<SubmissionViewModel>;
  public NoOfSubmissions: number;
  public TotalScore: number;
  public Position: number;
}

File: CodeFreak1/ViewModel/EventPerformanceViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class EventPerformanceViewModel:RequestStatus
    {
        public UsersViewModel User { get; set; }
        public FileViewModel UserImage { get; set; }
        public List<SubmissionViewModel> Submissions { get; set; }
        public int NoOfSubmissions { get; set; }
        public int TotalScore { get; set; }
        public int Position { get; set; }

    }
}
